opm/simulators/wells/MultisegmentWellEquations.cpp

```diff
@@ -402,6 +410,16 @@ extractCPRPressureMatrix(PressureMatrix& jacobian,
     }
 }
 
+template<class Scalar, int numWellEq, int numEq>
+void MultisegmentWellEquations<Scalar,numWellEq,numEq>::
+sumDistributed(Parallel::Communication comm)
```

Comment:
Remark (have not checked all code): This should either only do something or be called if there are more than one processes.